feat:add an file on how to make Django app #5

ankita0606 opened the pull request:
This PR adds an md file on how to create a Django App

ankita0606 committed:
feat:add an file on how to make Django app #5

animenon reviewed (commented):
Hi @ankita0606,

The markdown formatting seems to be off, do check.

ankita0606 commented:
Hi @animenon 
Thanks for the feedback. Will improve it ASAP.

ankita0606 committed:
Update creatingaDjangoApp.md

 I have corrected the markdown format. If any more changes are needed please let me know.

ankita0606 commented:
Hey @animenon ,
I have made the changes please go through them and let me know if any other changes are required.

animenon commented:
Hi @ankita0606 ,
The formatting still [looks off](https://github.com/animenon/Learning-Django/pull/26/files?short_path=e31126f#diff-e31126f655a97152d3545fa8d40dabd5).. Also maybe add the missing contents on an existing page? 
This file shows how to start a project, quite similar to [django_starter_app.md](https://github.com/animenon/Learning-Django/blob/django_examples/django_starter_app.md).

ankita0606 committed:
Update creatingaDjangoApp.md

animenon commented on creatingaDjangoApp.md:
````diff
@@ -0,0 +1,51 @@
+
+## Syntax for creating an app
+```
+Unix-python3 manage.py startapp APPNAME
+Windows-python manage.py startapp APPNAME
````
Add a newline after `Unix-` and `Windows-`

Also four spaces in front of the command for the formatting

animenon commented on creatingaDjangoApp.md:
````diff
@@ -0,0 +1,51 @@
+  ```
+## Example: Let's create an app which contains info of users called as Users
+```
+Unix-python3 manage.py startapp Users
+Windows-python manage.py startapp Users
````
Add a newline after `Unix-` and `Windows-`.

Also four spaces in front of the command for the formatting

animenon reviewed (commented):
Few minor changes.
Maybe merge this page with django_starter_app.md ?

animenon commented:
@ankita0606 planning to merge this file with the existing `django_starter_app.md`? 

Update quickly to be part of [hacktoberfest](https://hacktoberfest.digitalocean.com/).

ankita0606 committed:
combined with  django_starter_app.md

ankita0606 committed:
Update django_starter_app.md

ankita0606 committed:
Delete django_starter_app.md

ankita0606 committed:
combining files

ankita0606 committed:
Update django_starter_app.md

animenon commented:
Closing this as you have raised a new one #32 
You can push changes to the same branch so that new PR need not be raised.
